base_environment.yml

```diff
@@ -104,4 +104,4 @@ dependencies:
 # need to install from master until 0.10.1
 # 6762b474dd61f316bd952db0fa1d3ad96dc54174 is a known good commit but for now
 # not pinning the hash
-    - git+https://github.com/dask/dask-kubernetes
+    - git+https://github.com/dask/dask-kubernetes@master
```

Comment:
Not a big fan of depending on source in our images. Why not the last deployment?

Reply:
because the implementation of the remote scheduler is kind of wacky. I made a change (and one other person did too) that have been merged but not released yet. We have to make corresponding changes to `rhg_compute_tools` too, so I'd rather just make those now and pull from that source. I assume it will be merged soon, so I was only planning on this for test2 and/or compute-test and then hopefully having the new release ready by the time this is ready to merge into compute.rhg. We could pin the commit to pull from if we want?

Reply:
in fact I think we need to pin it to a commit, otherwise it keeps trying to pull from pypi (even if we give it the master branch). So I've updated that, but I still think we want to install from source until it has seen a new release (or we switch to dask-gateway)

Reply:
ok. what do you think is the timeline for this docker_images branch being stable?

Reply:
as soon as ppl start testing it! Unless we want to wait for dask-kubernetes to be re-released and/or for one of us to have time to port over to dask-gateway. But I'd say lets get people on it. Should we deploy it on compute-test?